src/paradigma/tremor/tremor_analysis_config.py

```diff
@@ -94,4 +94,26 @@ def __init__(self) -> None:
         self.d_channels_values["freq_peak"] = "Hz"
         self.d_channels_values["low_freq_power"] = "(deg/s)^2"
         self.d_channels_values["tremor_power"] = "(deg/s)^2"
-        
\ No newline at end of file
+        
+
+class TremorDetectionConfig(IMUConfig):
+
+    def __init__(self) -> None:
+        super().__init__()
+        self.coefficients_file_name = "tremor_detection_coefficients.txt"
+        self.thresholds_file_name = "tremor_detection_threshold.txt"
+        self.mean_scaling_file_name = "tremor_detection_mean_scaling.txt"
+        self.std_scaling_file_name = "tremor_detection_std_scaling.txt"
+
+        self.fmin_peak: float = 3
+        self.fmax_peak: float = 7
+        self.movement_treshold: float = 50.1
+
+        self.d_channels_values = {
+        "pred_tremor_proba": "probability",
+        "pred_tremor_logreg": "boolean",
+        "pred_tremor_checked": "boolean"
```

Comment:
Maakte je hier geen integers van? Ik denk dat opslaan als boolean misschien wel het beste is (neemt minder ruimte in).

Reply:
Nu zijn het inderdaad integers, scherp! Ik zal ze als boolean opslaan.